pallas-txbuilder/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "pallas-txbuilder"
+version = "0.19.0-alpha.1"
+edition = "2021"
+repository = "https://github.com/txpipe/pallas"
+homepage = "https://github.com/txpipe/pallas"
+documentation = "https://docs.rs/pallas-txbuilder"
+license = "Apache-2.0"
+readme = "README.md"
+authors = [
+  "Santiago Carmuega <santiago@carmuega.me>",
+  "Cainã Costa <me@cfcosta.com>"
+]
+
+[dependencies]
+hex = "0.4.3"
+minicbor = { version = "0.19.1", features = ["minicbor-derive"] }
```

Comment:
Agree, let me do that.

Reply:
Actually, it didn't work :(
What happens is: because of the way that the proc macros work, if you define `Encode` or `Decode` for a type, you need to have a `::minicbor` available, or it doesn't work. So whether or not I use it, I need to have `minicbor` on my Cargo.toml :(

Reply:
I see